content/en/observability_pipelines/guide/set_quotas_for_data_sent_to_a_destination.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Set Quotas for Data Sent to a Destination
+kind: documentation
+disable_toc: false
+private: true
+is_beta: true
+further_reading:
+- link: "/observability_pipelines/setup/"
+  tag: "Documentation"
+  text: "Set up Observability Pipelines"
+- link: "/observability_pipelines/working_with_data/"
+  tag: "Documentation"
+  text: "Working with data in Observability Pipelines"
+- link: "/monitors/configuration/"
+  tag: "Documentation"
+  text: "Learn more about configuring monitors"
+---
+
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSfnNnV823zAgOCowCYuXJE5cDtRqIipKsYcNpaOo1LKpGfppA/viewform" btn_hidden="false" header="Request Access!">}}
+The <code>quota</code> transform is in private beta.
+{{< /callout >}}
+
+## Overview
+
+The Observability Pipelines `quota` transform allows you to set a limit on the volume of data sent to a destination within a specific timeframe. This can safeguard you from unexpected data surges that might impact your operating costs. With the transform, you can set up different ways of handling the data when the quota has been reached. For example, you can:
+
+- Reroute the data sent after the quota limit to another destination, such as a cold storage.
+- Sample the data sent after the quota limit so that less data is sent to the destination.
+- Drop the data sent after the quota limit.
+- Set up a monitor to alert you that the quota has been reached.
+
+You can also use multiple `quota` transforms to set different warning and alert limits. For example, use the first `quota` transform to set a limit at a warning level and send a warning notification when the limit is reached. Then, use a second `quota` transform to set a hard limit. When the hard limit is reached, you can send out an alert notification and start sampling the data sent after the limit or reroute that data to another destination.
+
+This guide walks you through how to:
+
+- [Set up the quota transform](#set-up-the-quota-transform)
+- [Set up a monitor to alert when the quota is reached](#set-up-a-metric-monitor)
+- Route logs sent after the limit to `datadog_archives`
+
+## Set up the quota transform
+
+1. Navigate to [Observability Pipelines][1].
+1. Click on your pipeline.
+1. Click **Edit draft**.
+1. Click **+ Add Component**.
+1. Click the **Transforms** tab.
+1. Click the **Quota** tile.
+1. Enter a name for the component.
+1. Select the input(s) for the transform.
+1. In the **Limits** section:  
+    a. Select the unit type.  
+    b. Enter the limit in the **Max** field.
+1. Enter the timeframe in the **Window** field.  
+    For example, if you set:
+    - **Bytes** as the unit type  
+    - `2000000000`in the **Max** field  
+    - `24h` in the **Window** field  
+    Then the transform is configured to send up to 2GB of logs per day to the destination.
+1. Click **Save**.
```

Comment:
Let's add a final step on how to handle `[quota_id].dropped`. Let's start by explaining that data coming into the transform after the threshold is met gets routed to input ID of `[quota_id].dropped`, not `[quota_id]`. could be something like:

1. Click into a downstream component to decide where to route the data from the `quota` transform before _and_ after the quota is met. The input ID for data after the limit is met is `[quota_id].dropped`

Reply:
updated